pysap/plugins/mri/reconstruct/linear.py

```diff
@@ -24,7 +24,7 @@
 class Wavelet2(object):
     """ The 2D wavelet transform class.
     """
```

Comment:
Why changing the 2D wavelet transform?
Is there a bug?

Reply:
yes, without the kwargs we could not access to specific parameters of the wavelets other than those harcoded in the class definition (name and scales)